docs/src/examples.md

```diff
@@ -1,6 +1,84 @@
 # Examples
 
-## 1D Column examples
+### Advection
+
+The 1D Column advection example in [`examples/column/advect.jl`](https://github.com/CliMA/ClimaCore.jl/blob/main/examples/column/advect.jl).
+
+#### Equations and Discretizations
+
+Follows the continuity equation
+
+```math
+\begin{equation}
+  \frac{\partial \theta}{\partial t} = -\frac{\partial (v \theta)}{\partial z}
+\label{eq:1d-column-advection-continuity}
+\end{equation}
+```
+This is discretized using the following
+
+```math
+\begin{equation}
+  \frac{\partial \theta}{\partial t} \approx - D(V, \theta) 
+\label{eq:1d-column-advection-discrete}
+\end{equation}
+```
+
+#### Prognostic Variables
+
+* ``\theta``: the scalar field
+* ``v``: the velocity field
+
+#### Tendencies 
+
+The example code solves the equation for 4 different tendencies with the following discretizations:
+
+* Tendency 1: ``D = \partial(UB),`` where ``\partial`` is the [`face-to-center divergence`](https://clima.github.io/ClimaCore.jl/dev/operators/#ClimaCore.Operators.DivergenceF2C) and $UB$ is the [`center-to-face upwind product operator`](https://clima.github.io/ClimaCore.jl/dev/operators/#ClimaCore.Operators.UpwindBiasedProductC2F)
+* Tendenct 2: Follows tendency 1 with the addition of flux correction `fcc`
```

Comment:
Please describe in the formula and put a link to the `FluxCorrectionC2C` operator as you did for Tendency 1 here